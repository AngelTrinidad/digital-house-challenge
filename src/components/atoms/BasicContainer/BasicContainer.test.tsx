import React from 'react';
import {Text} from 'react-native';

import {render} from '~helpers/testing';

import BasicContainer from '.';

describe('<BasicContainer />', () => {
  let component: any;

  beforeEach(() => {
    component = render(
      <BasicContainer>
        <Text>Hello</Text>
      </BasicContainer>,
    );
  });

  it('renders correctly', () => {
    expect(component).toBeDefined();
  });

  it('has same snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
