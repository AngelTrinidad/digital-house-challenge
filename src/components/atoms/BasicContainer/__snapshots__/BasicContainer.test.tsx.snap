// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasicContainer /> has same snapshot 1`] = `
<View
  style={
    [
      {
        "flex": 1,
      },
      undefined,
      [
        {
          "padding": 20,
        },
        {
          "backgroundColor": "#F8F8F8",
        },
        undefined,
      ],
    ]
  }
>
  <Text>
    Hello
  </Text>
</View>
`;
